testing/TestHarness/TestHarness.Shared/Ext/Validation/ValidationHostInit.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Uno.Extensions.Validation;
+using TestHarness.Ext.Navigation.Validation;
+
+namespace TestHarness;
+
+public class ValidationHostInit : BaseHostInitialization
+{
+	protected override IHostBuilder Custom(IHostBuilder builder)
+		=>builder
+			.UseValidation(configure:
+			builder=>
```

Comment:
the variable "builder" is used twice - change the name of one of them to make it clearer